build.sbt

```diff
@@ -138,6 +180,7 @@ lazy val monitoring =
       libraryDependencies += "com.typesafe.akka" %% "akka-http-experimental" % akkaVersion, // FIXME: shadow that dependency and hide it
       libraryDependencies += "org.scalatest" %%% "scalatest" % scalatestVersion % "test",
       libraryDependencies += "io.getquill" %% "quill-cassandra" % "0.4.1",
+      libraryDependencies ++= servedAssets,
```

Comment:
Is it needed here? Those assets are bundled as single .js file, is it needed for css?